e2e/quick_start_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("e2e quickstart", Ordered, func() {
+	exampleName := "quick-start"
```

Comment:
Maybe we should place the example in testdata/.

Reply:
LGTM

Reply:
I think I could move the extra `envd-quick-start` to testdata. And other tests shoule be wrote to cover the examples

Reply:
examples may be complex. Personally, I prefer to test different features in different test cases. WDYT

Reply:
LGTM

Reply:
what do you mean ` then update the CI.yaml?`